### DISCUSSION TITLE
Release plan

### DISCUSSION THREAD

**Discussion by itwolf81** [LGouellec/streamiz#261] [Q&A] [answered]
Hello, thank you for your great work on this project. Do you have some release plan page, where I can see which feature is planned in next release and when do you plan to have next release? Thank you.

**Comment by LGouellec** [comment, marked as answer]
Hi @itwolf81 , 
I don't have any release page for now. I used "tags" on the issue to track the different milestone. 
And you can see the progress of the current milestone [here](https://github.com/LGouellec/kafka-streams-dotnet/milestones).

If you have any question, or any feature request. Don't hesitate !

**Reply by itwolf81** [reply]
Thanks, I was just curious, what features are in progress on this project. I did successfully my POC for our team using your nuget. From my findings:
- it was quite hard to find right configuration regarding kafka user permissions. We are using sasl with very strict permission list for users on our kafka cluster. Your logic requires these from my findings:
```    acls:
      - resource:
          type: group
          name: # requires the same name as kafka stream application.id
          patternType: prefix
        operations:
          - Read
          - Describe
      - resource:
          type: cluster
          name: '*'
          patternType: literal
        operations:
          - DescribeConfigs
      - resource:
          type: topic
          name: # source kafka topic for the stream
          patternType: literal
        operations:
          - Read
          - Describe
        host: "*"
      - resource:
          type: topic
          name: # target kafka topic for the stream
          patternType: literal
        operations:
          - Write
          - Describe
      - resource:
          type: topic
          name: [application.id]- # state store topics created by streamiz nuget
          patternType: prefix
        operations:
          - Read
          - Write
          - Describe
          - Create
          - Delete
```
- RocksDb requires other nuget (RocksDb.Native) to run in kubernetes. I'm still not sure what are all the required steps to make it running in k8s. So currently I'm using InMemory store.

Hope that it helps for someone.
